content/influxdb/v2.0/reference/cli/influxd/upgrade.md

```diff
@@ -49,6 +49,7 @@ influxd upgrade [command]
 |      | `--log-level`                   | Supported log levels are `debug`, `info`, `warn`, and `error` (default `info`)                             | string     |
```

Comment:
Can't comment on it directly, but `--influx-command-path` above is no longer a supported CLI option.